plugin-gradle/src/main/java/com/diffplug/gradle/spotless/KotlinGradleExtension.java

```diff
@@ -104,6 +105,47 @@ private FormatterStep createStep() {
 		}
 	}
 
+	/** Adds the specified version of [diktat](https://github.com/cqfn/diKTat). */
+	public DiktatFormatExtension diktat(String version) {
+		Objects.requireNonNull(version, "version");
+		return new DiktatFormatExtension(version, Collections.emptyMap(), null);
+	}
+
+	public DiktatFormatExtension diktat() {
+		return diktat(DiKTatStep.defaultVersionDiktat());
+	}
+
+	public class DiktatFormatExtension {
```

Comment:
What's the difference between `KotlinExtension` and `KotlinGradleExtension`?

Reply:
Oh, I see the latter is some sort of special extension for kotlin scripts, specifically from gradle. Ok, that's weird, but Ok.